aiodataloader.py

```diff
@@ -1,29 +1,82 @@
-from asyncio import gather, ensure_future, get_event_loop, iscoroutine, iscoroutinefunction
-from collections import namedtuple
-from collections.abc import Iterable
+from asyncio import (
+    AbstractEventLoop,
+    Future,
+    gather,
+    ensure_future,
+    get_event_loop,
+    iscoroutine,
+    iscoroutinefunction,
+)
 from functools import partial
 
-from typing import List  # flake8: noqa
+from typing import (
+    Any,
+    Callable,
+    Coroutine,
+    Dict,
+    Generic,
+    Iterable,
+    Iterator,
+    List,
+    Optional,
+    TypeVar,
+    Union,
+)
+from typing_extensions import Protocol, TypeGuard
+
+__version__ = "0.2.1"
```

Comment:
nitpick -- this PR really shouldn't make any changes that are not required to add typing, so that its really clear what this change does.

Reply:
Back to single quotes it is. I'm more used to working in codebases that are already `black`ed and this one snuck through on my `git add -p` removing the `black` artifacts from before, so I won't take any of these nits personally.